Source/Core/loadImageFromTypedArray.js

```diff
@@ -31,22 +31,29 @@ define([
             type : format
         });
 
-        var blobUrl = window.URL.createObjectURL(blob);
-        var resource = new Resource({
-            url: blobUrl,
-            request: request
-        });
-        return resource.fetchImage({
-            flipY : flipY,
-            preferImageBitmap : true
-        })
-            .then(function(image) {
-                window.URL.revokeObjectURL(blobUrl);
-                return image;
-            })
-            .otherwise(function(error) {
-                window.URL.revokeObjectURL(blobUrl);
-                return when.reject(error);
+        return Resource.supportsImageBitmapOptions()
+            .then(function(result) {
+                if (result) {
+                    return when(Resource.createImageBitmapFromBlob(blob, flipY));
+                }
+
+                var blobUrl = window.URL.createObjectURL(blob);
+                var resource = new Resource({
+                    url: blobUrl,
+                    request: request
+                });
+
+                return resource.fetchImage({
+                    flipY : flipY
+                })
+                .then(function(image) {
+                    window.URL.revokeObjectURL(blobUrl);
+                    return image;
+                })
+                .otherwise(function(error) {
```

Comment:
You can replace the last two `then/otherwise` calls here with a single `always` call that does not take or return a value and just calls `window.URL.revokeObjectURL(blobUrl);`